Gemfile.next

```diff
@@ -0,0 +1 @@
+Gemfile
```

Comment:
This is apparently what symlinks look like in Github's diffs. So note, this file is a symlink to the normal gemfile, not a file with this line in it.